src/main/java/com/team2502/robot2023/subsystems/DrivetrainSubsystem.java

```diff
@@ -379,6 +389,17 @@ public double getHeading() {
         return navX.getAngle();
     }
 
+    public void resetPitch() {
```

Comment:
Recommend renaming to "resetRoll" to reduce confusion in naming

Reply:
resolved by [15b60](https://github.com/Team-2502/RobotCode2023/pull/9/commits/15b60829f4d6390c5c0c9d459b2de9871e215676)